modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java

```diff
@@ -2821,6 +2867,7 @@ at the moment path passed as string (basePath) from upper components can be both
 			if (!specKeys.get("SCHEMA_KEYS").contains(key) && !key.startsWith("x-")) {
 				result.extra(location, key, node.get(key));
 			}
+			validateReservedKeywords(specKeys, key, location, result);
```

Comment:
This is handling 3.0, so not applicable. Also I believe this restriction is not applied to Json Schema also in 3.1

Reply:
I'll remove the validation from there but I think in the method we'll handle 3.1 schema(JsonSchema) we may find also extensions with `x-` or without.  
`This object MAY be extended with Specification Extensions, though as noted, additional properties MAY omit the x- prefix within this object.`